api/src/main/java/jakarta/data/page/CursoredPage.java

```diff
@@ -120,7 +120,7 @@
  * <p>For example:</p>
  *
  * <pre>
- * &#64;Query("SELECT o FROM Customer o WHERE (o.ordersPlaced &gt;= ?1 OR o.totalSpent &gt;= ?2)")
+ * &#64;Query("WHERE ordersPlaced &gt;= ?1 OR totalSpent &gt;= ?2")
```

Comment:
Oh, yes, it is still there:

> Cursor-based pagination involves generating and appending additional restrictions involving the key fields to the WHERE clause of the query. For this to be possible, a user-provided JDQL or JPQL query must end with a WHERE clause to which additional conditions may be appended without otherwise changing the semantics of the query:

I thought it was gone.

[And, FTR, I think it *should* go. We don't have subqueries in JDQL, so the task of inserting parens in between `where` and `order by` is just completely trivial.]